WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryViewModel.cs

```diff
@@ -69,7 +71,10 @@ public void SelectTransaction(uint256 txid)
 			if (txnItem is { })
 			{
 				SelectedItem = txnItem;
-				SelectedItem.IsFlashing = true;
+				if (SelectedItem is { })
```

Comment:
We don't need this check because `txnItem` isn't null.

Reply:
I think there is possibility for SelectedItem property to be set to null even after setting to txnItem

Reply:
But isn't there a possibility for `SelectedItem` property to be set to null after we check it is not null?

Reply:
@yahiheb is right, there is no chance for the SelectedItem to be null.

Reply:
Reverted changes